x-pack/plugins/enterprise_search/public/applications/workplace_search/constants.ts

```diff
@@ -734,6 +734,10 @@ export const FIELD_LABEL = i18n.translate('xpack.enterpriseSearch.workplaceSearc
   defaultMessage: 'Field',
 });
 
+export const LABEL_LABEL = i18n.translate('xpack.enterpriseSearch.workplaceSearch.label.label', {
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
export const LABEL_LABEL = i18n.translate('xpack.enterpriseSearch.workplaceSearch.labelLabel', {
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697